README.md

```diff
@@ -421,7 +489,11 @@ The SDK provides `IKUpload` component to upload files to the ImageKit Media Libr
 | isPrivateFile | Boolean | Optional. Accepts `true` of `false`. The default value is `false`. Specify whether to mark the file as private or not. This is only relevant for image type files|
 | customCoordinates   | String | Optional. Define an important area in the image. This is only relevant for image type files. To be passed as a string with the `x` and `y` coordinates of the top-left corner, and `width` and `height` of the area of interest in format `x,y,width,height`. For example - `10,10,100,100` |
 | responseFields   | Array of string | Optional. Values of the fields that you want upload API to return in the response. For example, set the value of this field to `["tags", "customCoordinates", "isPrivateFile"]` to get value of `tags`, `customCoordinates`, and `isPrivateFile` in the response. |
-| onSuccess   | Function callback | Optional. Called if the upload is successful. The first and only argument is the response JOSN from the upload API |
+| inputRef   | Reference | Optional. Forward reference to the core HTMLInputElement.|
+| onUploadStart | Function callback | Optional. Called before upload is started. The first and only argument is the HTML input's change event |
```

Comment:
This event is fired regardless of actual start of upload because this is essentially a change event. However if `validateFile` returns `false` and upload never starts then this event should not be fired.